src/main/java/baseball/Model/GameResult.java

```diff
@@ -0,0 +1,44 @@
+package baseball.Model;
+
+import java.util.List;
+
+public class GameResult {
+
+    private int ball = 0;
+    private int strike = 0;
```

Comment:
이렇게 필드에서 생성과 동시에 초기화하는 방법이 아닌, 생성자를 통한 초기화 방법은 어떨까요? 
```suggestion
    private int ball;
    private int strike;
    
    public void GameResult(){
        this.ball = 0;
        this.strike = 0;
    }
```